src/figuresClasses.ts

```diff
@@ -1,19 +1,80 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public sideA: number,
+    public sideB: number,
+    public sideC: number,
+  ) {
+    if (sideA <= 0 || sideB <= 0 || sideC <= 0) {
+      throw new Error('No one side cannot be less than or equal to 0');
+    }
+
+    const sides: number[] = [sideA, sideB, sideC].sort(
+      (current, next) => next - current,
+    );
+
+    if (sides[0] >= sides[1] + sides[2]) {
+      throw new Error(`sides ${sideA}, `
+        + `${sideB} and ${sideC} can't form a triangle`);
+    }
+  }
 
-export class Circle {
+  getArea(): number {
+    const p: number = (this.sideA + this.sideB + this.sideC) / 2;
 
+    return Math.floor(
+      Math.sqrt(p * (p - this.sideA) * (p - this.sideB) * (p - this.sideC))
+      * 100,
+    ) / 100;
+  }
```

Comment:
the formatting looks wrong a little. Think of sth to make it more readable

Reply:
I've made changes acceptable for eslint, please, take a look.